### PR TITLE
Translate tool into English

### PR DESCRIPTION
**Pull request by tobijat**

### CODE REVIEW

**Comment by manicki** [comment]
For the record (as I was wondering what this is about): English translations are going to be added to this PR's branch. Once all is translated, this could be merged. No other needed here.

File: i18n/en/i18n.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    "editing­question­mark": "All changes of the image that alter its impression on its viewers (like clipping, coloring, drastic re­sizing etc.) make the result an adaptation in the legal sense. In contrast, changing the image’s file format or digitizing it does not result in adaptations because the image and the impression it gives remain the same.",
+    "licence­question­mark": "The tool only offers licenses which are compatible with the Creative Commons license attached to the original work.",
+    "creator­question­mark": "Please state, if possible, your full name. A pseudonym, a nickname or “anonymous” is possible as well.",
+    "same­licence": "Same license as the original work.",
+    "licence­text": "License text",
+    "download­image­link": "Downlowad the image",
```
Comment (manicki) [inline i18n/en/i18n.json]: Download

File: i18n/en/legal.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+</p>
+<p>
+	This website uses Piwik, an Open Source software for statistical evaluation and analysis of page visits. Piwik uses cookies. The cookie created by Piwik contains information about your use of the website and are saved on our servers in Germany. The IP address is, after its processing, directly anonymized. You can avoid the creation of cookies via your browser settings. However, we advise you that you might not be able to use all functions of the website. Furthermore, you have the option to decline any data process by Piwik for the future. Simply disable the check box below the text. By doing so, Piwik creates a Piwik deactivation cookie in your browser. Please note that this cookie will be deleted, if you delete all cookies in your browser.
+</p>
+<p>
+	You can decide here if this tool is allowed to create a web analysis cookie in your browser, which helps us to evaluate and analyse different data of your page visit.
+    If you are against if, please follow the link to create an a Piwik deactivation cookie in your browser.
```
Comment (WMDE-Fisch) [inline i18n/en/legal.html]: 'If you are against **it**...."

File: i18n/en/i18n.json [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    "send­feedback": "Something did not work. Please try again.",
+    "feedback­generic": "Something did not work. Please try again.",
+    "feedback­blank­fields": "Please fill in all the fields of the feedback form."
+  },
+  "info­box": {
+    "private­use": "For most purely private uses, you do not need a license notice.",
+    "private­use­more­link": "&#8594; LEARN MORE",
+    "dont­show­again": "Don’t show this message again.",
+    "understood­and­close": "Understood and close.",
+    "ported­licence": "Certain Creative Commons licenses were published with country-specific editions, so­called “ported licenses”. The image you were asking a license notice for, was published with such a ported license. The tool is not able to check whether the license requirements of this country­specific Creative Commons license are equal to the version in English language. However, you can still generate a license notice attribution with this tool. To be entirely sure, we recommend to check the license requirements of this country­specific version. Additionally, the license notice has to be in the language of its context to fulfill its function completely."
```
Comment (manicki) [inline i18n/en/i18n.json]: so-called

**Comment by manicki** [comment]
Fixed yet another typo. Could be merged after https://github.com/wmde/Lizenzhinweisgenerator/pull/220 gets in

**Pull request merged**